common/result.c

```diff
@@ -118,6 +118,12 @@ const char* oe_result_str(oe_result_t result)
             return "QE_QUOTE_ENCLAVE_IDENTITY_PRODUCTID_MISMATCH";
         case OE_VERIFY_FAILED_AES_CMAC_MISMATCH:
             return "OE_VERIFY_FAILED_AES_CMAC_MISMATCH";
+        case OE_SWITCHLESS_OCALL_MISSED:
```

Comment:
If this error code is needed at all, then rename to OE_CONTEXT_SWITCHLESS_OCALL_MISSED